netbox_branching/models/changes.py

```diff
@@ -48,14 +48,18 @@ def apply(self, branch, using=DEFAULT_DB_ALIAS, logger=None):
 
         # Creating a new object
         if self.action == ObjectChangeActionChoices.ACTION_CREATE:
-            instance = deserialize_object(model, self.postchange_data, pk=self.changed_object_id)
+            if hasattr(model, 'deserialize_object'):
```

Comment:
Tiny edge-case here, it's possible that some model, somewhere (in core in the future or a plugin model that nominally supports branching) has a `deserialize_object` attr that is not callable.

I'm only calling this out because I could see a random plugin doing something like this and it costing us a fair amount of time to debug and respond.